docs/user-guide/import-certificates.md

```diff
@@ -0,0 +1,176 @@
+# Import and configure a certificate if you have one
+
+One option for using certificates in Zowe is to import and configure existing certificates. Zowe supports the following types of certificates for import:
+
+* File-based [PKCS12 certificates](#import-an-existing-pkcs12-certificate)
+* [JCERACFKS certificates](#import-an-existing-jceracfks-certificate)
```

Comment:
Or more? Not sure if we've claimed support for other types of keyrings yet.
Though either way - do we actually want to say KCERACFKS here, or just say z/OS keyring